hr_phone/models/hr_employee.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2012-2021 Akretion France (http://www.akretion.com/)
+# @author: Alexis de Lattre <alexis.delattre@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class HrEmployeePrivate(models.Model):
```

Comment:
As I commented in the `base_phone` PR, I don't think this `phone.validation.mixing` makes sense anymore.
Use `odoo.addons.phone_validation.tools.phone_format` instead.